resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends('auth.layouts.master')
+@section('content')
+    <div class="container-fluid px-1 px-md-5 px-lg-1 px-xl-5 py-5 mx-auto">
+        <div class="d-flex justify-content-end">
+            <ul>
+                <li style="padding: 0px 3px" class="{{\Illuminate\Support\Facades\App::getLocale()=='en'?'active':''}}">
```

Comment:
```suggestion
                <li class="pr-3 {{ app()->getLocale() == 'en' ? 'active' : '' }}">
```
Ko viết style vào trong html trực tiếp. Tất cả css viết ra file riêng nhé :)